index.html

```diff
@@ -155,6 +157,83 @@ <h2>
         </p>
       </div>
     </section>
+    <section> <h3>Permissions and user prompts</h3>
+      <p>
+        The term <a>obtain permission</a>
+        is used for acquiring trust for a given operation.
+      </p>
+      <p>
+        The term <dfn>expressed permission</dfn> refers to an act by the user, e.g.
+        via user interface or setting or host device platform features, through which
+        the user approves the permission of a <a>browsing context</a> to access the
+        given functionality.
+      </p>
+      <p>
+        All <a>expressed permission</a>s that are preserved beyond the current
+        browsing session MUST be revocable.
+      </p>
+      <p>
+        The term <dfn id="askforgiveness">ask for forgiveness</dfn> refers to some
+        form of unobtrusive notification that informs the user of an operation
+        while it is running.
+      </p>
+      <p>
+        UAs SHOULD provide the user with the means to block the ongoing operation
+        from the current <a>origin</a>, or simply dismiss the notification,
+        and the choice should be remembered.
+      </p>
+      <p>
+        The term <dfn>prearranged trust relationship</dfn> means that the
+        UA has already established a trust relationship for a
+        certain operation using a platform specific mechanism, so that an
+        <a>expressed permission</a> from the user is not any more needed.
+      </p>
+      <p>
+        The term <a>obtain permission</a> for a certain operation indicates
+        that the UA has either obtained <a>expressed permission</a>, and/or
+        <a href="#askforgiveness">asked for forgiveness</a>, and/or ensured a
```

Comment:
This contradicts the `<dfn>obtain permission</dfn>` steps.

Reply:
I am removing this and rewriting it